build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.2'
+        classpath 'com.android.tools.build:gradle:2.3.3'
```

Comment:
1. Is it absolutely necessary to upgrade this version?
2. I am assuming developers will need to upgrade these tools for their development, is this right?
3. Will the end users need to do any android updates or system updates in order to run this new app?